src/Be.Vlaanderen.Basisregisters.Api/StartupDefaults-Use.cs

```diff
@@ -272,8 +272,7 @@ public static partial class StartupDefaults
             });
             options.MiddlewareHooks.AfterSwagger?.Invoke(app);
 
-            app.UseMvc();
-            options.MiddlewareHooks.AfterMvc?.Invoke(app);
```

Comment:
We kept it to `AfterMvc` because it was AfterMvc before they used `UseMvc`.
If you want to rename it for consistency then I would add 2 hooks `AfterRouting` and `AfterEndpoints` after each `UseXXX`